pkg/version/version.go

```diff
@@ -37,18 +36,39 @@ var (
 	Version string //nolint:gochecknoglobals
 	// FullCommit is a git commit hash.
 	FullCommit string //nolint:gochecknoglobals
-	// CatalogImage is a image path for OLM catalog.
-	catalogImage string //nolint:gochecknoglobals
 )
 
-// CatalogImage returns a catalog image needed for the build of everestctl.
-func CatalogImage() string {
-	catalogImage = devCatalogImage
-	v, err := goversion.NewSemver(Version)
-	if Version != "" && err == nil && v.Prerelease() == "" {
-		catalogImage = fmt.Sprintf(releaseCatalogImage, Version)
+// CatalogImage returns a catalog image name.
+func CatalogImage(v *goversion.Version) string {
+	if isDevVersion() {
+		return devCatalogImage
 	}
-	return catalogImage
+	return fmt.Sprintf(releaseCatalogImage, v)
+}
+
+// ManifestURL returns a manifest URL to install Everest.
+func ManifestURL(v *goversion.Version) string {
+	if isDevVersion() {
+		return devManifestURL
+	}
+	return fmt.Sprintf(releaseManifestURL, v)
```

Comment:
there is no such constants in the codebase anymore. the latest change was using the embed manifest that pointed out to the current version.

Reply:
Seems it got lost during migration. Added it back. Thank you for pointing it out.